src/runtime.js

```diff
@@ -0,0 +1,22 @@
+import sdkVersion from './version';
+
+/* global window, navigator, process */
+const isBrowser = typeof window !== 'undefined' && typeof window.document !== 'undefined';
+const navigatorUserAgent = typeof navigator !== 'undefined' ? navigator.userAgent : '';
+const nodeVersion =
+  typeof process !== 'undefined' && typeof process.versions !== 'undefined'
```

Comment:
Couldn't this be checked as `process && process.versions`?

Reply:
We use the more strict checks in a bunch of places, so I guess that's been preferred in the past.